cinn/hlir/op/broadcast.cc

```diff
@@ -377,6 +377,7 @@ std::vector<std::vector<std::string>> InferLayoutForIsClose(const std::vector<st
 }
 
 StrategyForBinary(elementwise_add, Add);
+StrategyForBinary(elementwise_atan2, Atan2);
```

Comment:
你这名称还是叫`elementwise_atan2`呀。。。CI当然会过不了，这儿也得改成`atan2`：
```suggestion
StrategyForBinary(atan2, Atan2);
```

Reply:
好的好的，已修改，忘记改这里了